assets/settings/default.json

```diff
@@ -198,7 +198,12 @@
     // Where to dock channels panel. Can be 'left' or 'right'.
     "dock": "left",
     // Default width of the channels panel.
-    "default_width": 240
+    "default_width": 240,
+    // Whether to show the git icon in the title bar.
+    // This setting is only relevant when the git branch is enabled.
+    "git_icon": false,
```

Comment:
These settings don't really belong beneath `collaboration_panel`, as they are not related to collaboration.

We probably need a new `title_bar` section, or maybe these could go under our existing `git` settings?

Reply:
I'm going to put them in git section. Makes sense and also follows suit with other patterns